pkg/disttask/framework/taskexecutor/task_executor_test.go

```diff
@@ -415,7 +380,7 @@ func TestTaskExecutor(t *testing.T) {
 	// no-subtask check counter should be reset after a subtask is run.
 	mockStepExecutor.EXPECT().Init(gomock.Any()).Return(nil)
 	mockSubtaskTable.EXPECT().GetSubtasksByExecIDAndStepAndStates(gomock.Any(), "id", taskID, proto.StepOne,
-		unfinishedNormalSubtaskStates...).Return(nil, nil)
+		unfinishedNormalSubtaskStates...).Return(nil, nil).AnyTimes()
```

Comment:
the mock of GetSubtasksByExecIDAndStepAndStates in checkBalanceSubtask is at line 316, why change it

Reply:
revert it 2d586c05f513d698c63f4349410fcd3202686e1d